lava_test_plans/testcases/android-cts-lkft.yaml

```diff
@@ -0,0 +1,26 @@
+{% extends "testcases/android-cts.yaml" %}
+
+{% set test_timeout = 480 %}
+{% set test_name = "cts-lkft" %}
```

Comment:
what makes this a lkft job?

Reply:
not get your point, why do you think it is not a lkft job?

Reply:
what makes these jobs lkft specific or can 'lkft' be dropped from the test_name ?

Reply:
As you could see there are a list of modules specified here, it's called the cts-lkft test plan, which is specific to the lkft project as it's defined by the lkft project,

Reply:
So, 1, random cts include-filters enabled that we decided are good to enable, cts-lkft isn't a describing name for this testcase. lkft in the testcase at all isn't describing for what the test does.
lkft is a project not a testcase.

2. that you have cts-lkft as a test_name in two different test will mess up the results in LAVA right? Two suites with the same name or?

Reply:
1. this test case and others are only for the lkft project, and cts-lkft is currently used to described that since the beginning, lkft is a project test, but why it could not be used to define the testcase? 
2. yes, which is required to find the history path in squad automatically.

Reply:
you are correct.

However, its not needed to have cts-lkft in the job-name, since if you look at how that is built up there is already project which contains lkft-android...

And the filename should probably not be named android-cts-lkft.yaml either.
Yes its the lkft project that have defined this set of include-filters in some way.
Either add a more descriptive name or maybe call it cts-smoke since that's what it looks like.
Since you said that you will add more cts tests, like android-cts-part1, android-cts-part2, android-cts-part3... android-cts-partN.

Reply:
cts-lkft is the best option for both the job name and the file name here.
smoke is a generic name that should be able to be used for any projects,
This is the cts-lkft plan, for the lkft project, other projects could use it, but it's still the cts-lkft test plan.